src/commands/explore.rs

```diff
@@ -32,8 +32,10 @@ impl ExploreCommand {
 
         let config = self.common.config(self.target.as_deref())?;
 
-        let wasm = std::fs::read(&self.module)
+        let bytes = std::fs::read(&self.module)
             .with_context(|| format!("failed to read Wasm module: {}", self.module.display()))?;
+        let wasm = wat::parse_bytes(&bytes)
+            .with_context(|| format!("failed to parse Wasm module: {}", self.module.display()))?;
```

Comment:
Can you make sure that this also compiles and works for wasm binaries when the `wat` dependency isn't enabled? Something like the following:

```rust
                #[cfg(feature = "wat")]
                let bytes = wat::parse_bytes(bytes).map_err(|mut e| {
                    e.set_path(path);
                    e
                })?;
```

Reply:
Sorry I forgot `wat` was behind a feature. Fixed it now.